config/public-testnet.yaml

```diff
@@ -131,21 +131,19 @@ checks:
   pt-testnet-load:
     options:
       content-size: 50000000
-      postage-amount: 1000000000 # 1024/15*60*60*24*7
+      postage-amount: 1428084000 # (28335/12)*60*60*24*7
       postage-depth: 24
-      duration: 12h
+      duration: 24h
```

Comment:
why not even longer

Reply:
feel free to suggest any values? When we deploy it as cron job, it will be restarted in any case?

Reply:
make it a month so we dont see so many pods created in k9s :)

Reply:
To increase the postage-amount as well?

Reply:
To increase the postage-amount as well? To not cause some overspending?

Reply:
done